src/types/types.d.ts

```diff
@@ -96,6 +96,19 @@ export interface KeyStoreInitialState {
   };
 }
 
+export interface LiquidityPoolInitialState {
+  data: LiquidityPoolAccountTransaction[];
+  hasMoreTxs?: boolean;
+  status: ActionStatus | undefined;
+  errorString?: string;
+}
+
+export interface ClaimableBalancesInitialState {
+  data: ClaimableBalance[];
+  status: ActionStatus | undefined;
+  errorString?: string;
```

Comment:
in both of these, shouldn't it be

`errorString: string | undefined`?

Looks like the initial state defines `errorString`, so the key should always be present instead of optional

Reply:
That is a good point, we have done it this way in all other ducks. I'd prefer the more strict way and will create a [separate ticket](https://github.com/stellar/account-viewer-v2/issues/345) to update all for consistency.